.github/workflows/e2e-main.yml

```diff
@@ -30,12 +30,9 @@ jobs:
     - name: Prepare back-end environment
       uses: ./.github/actions/prepare-backend
 
-    - run: ./bin/build version
-    - run: ./bin/build translations
-    - run: ./bin/build frontend
-    - run: ./bin/build licenses
-    - run: ./bin/build drivers
-    - run: ./bin/build uberjar
+    - name: Build uberjar with ./bin/build
+      run: >-
+        ./bin/build version translations frontend licenses drivers uberjar
```

Comment:
This is not directly related to this fix, but I noticed this while investigating. You can run all these steps in one command instead of six separate commands. This should speed up CI a lot since it we only have to pay the Clojure startup time penalty once instead of six times